lib/elixir/lib/kernel.ex

```diff
@@ -5462,6 +5462,46 @@ defmodule Kernel do
     raise ArgumentError, "modifier must be one of: s, a, c"
   end
 
+  @doc """
+  Handles the `~PID` sigil.
+  """
```

Comment:
We should probably add a note about hardcoding PID values. Same for Port and Reference. The idea is that you can use these sigils for copy and pasting while debugging, but PIDs are not guaranteed to be the same between VM runs.

Reply:
Good call.

list_to_pid docs say [1]

> This BIF is intended for debugging and is not to be used in application programs.

we have a similar warning for Function.info/1:

> **Note**: this function must be used only for debugging purposes.

should we just use that or talk a little bit more about the VM as you mentioned?

[1] http://erlang.org/doc/man/erlang.html#list_to_pid-1
[2] https://hexdocs.pm/elixir/Function.html#info/1

Reply:
Your call. The clearer we can be, the better.

Reply:
Done!